stable/characteristics.md

```diff
@@ -1 +1,33 @@
-coming soon...
\ No newline at end of file
+# Device characteristics
+
+## Goal
+
+* 15 frames per second
+* 128 gray level
+* 64-128 lines
+* 60° sector image
+* 1 mm resolution
+* 3 frequencies:
+* * 3.5 MHz (focal from 50 to 150 mm)
+* * 5 MHz (focal from 30 to 105 mm)
+* * 7.5 MHz (focal from 10 et 75 mm)
+
+## Actually reach
+
+* around 3 frames per second
+* 128 gray level
+* 64-128 lines
+* 60° sector image
+* 1 mm radial resolution
+* lateral resolution is unknowed yet
+* 1 frequency (3.5 MHz)
+
+## Informations
```

Comment:
This section should include the following pieces of information :
- recall the default tunings of the device --> There should be a **subsection "Default tunings"** where you recall these informations
- depth of detection, angular coverage --> the paragraph about depth of detection is not clear to me ? + these informations should be in a **subsection "Detection characteristics"**
- SNR --> can you quantify this + add the nice figures you made to visualize the last improvement? --> in a **subsection "Raw signal characteristics"**
- number of pixels, size of pixels, framerate --> can you gather those informations together with the image size, in a unique **subsection "Display characteristics"** ? 
- add a screenshot of a beautiful echOpen image :-)